docs/src/components/NavLink.js

```diff
@@ -26,6 +27,7 @@ const NavLink = ({ children, to, history }: Props) => {
     if (isModifiedEvent(event) || !isLeftClickEvent(event)) return;
     event.preventDefault();
     history.push(to);
+    if (scrollToTop) window.scrollTo(0, 0);
```

Comment:
Hey Igor, I think we can just leave here, no need for the `scrollToTop` prop.

`window.scrollTo(0, 0);`

Every time we click, it scrolls to top.

Reply:
@AlbertCarreras thank you for the tip and for the quick review! I updated it, please let me know if you think other changes are required.